kDriveCore/Data/Models/Drive/Drive.swift

```diff
@@ -60,13 +60,13 @@ public enum MaintenanceReason: String, PersistableEnum, Codable {
     case technical
 }
 
-public class DrivePreferences: EmbeddedObject, Codable {
+public final class DrivePreferences: EmbeddedObject, Codable {
     @Persisted public var color = "#0098FF"
     @Persisted public var hide = false
 }
 
-public class Drive: Object, Codable {
-    @Persisted(primaryKey: true) public var objectId = ""
+public final class Drive: Object, Codable {
+    @Persisted(primaryKey: true) public var objectId: String
```

Comment:
Realm doesn't seem happy if the default value isn't provided directly here

Reply:
Discussed and modified.